web/src/views/Apply/FormQuestions.tsx

```diff
@@ -62,14 +61,15 @@ export const FormQuestions = () => {
               <TitledField name="firstName" label="Nombres *"/>
               <TitledField name="lastName" label="Apellidos *"/>
               <TitledField name="email" label="Correo electrónico *"/>
-              <TitledField name="institution" label="Centro de estudios *"/>
+              <TitledField name="institution" label="Centro de estudios"/>
             </div>
             <div className="grid grid-cols-1 gap-y-4">
               <h1 className="text-3xl font-bold mt-5">Preguntas</h1>
               <div>
-                <h2>Link personal *</h2>
+                <h2>Link personal</h2>
```

Comment:
I think we may want to pluralize this.
```suggestion
                <h2>Links personales</h2>
```